src/momento-redis-adapter.ts

```diff
@@ -110,29 +110,23 @@ export interface MomentoIORedis {
 
   pexpire(key: RedisKey, milliseconds: number): Promise<number | null>;
 
-  pexpire(
-    key: RedisKey,
-    milliseconds: number,
-    nx: 'NX'
-  ): Promise<number | null>;
+  pexpire(key: RedisKey, milliseconds: number, nx: 'NX'): Promise<number>;
 
-  pexpire(
-    key: RedisKey,
-    milliseconds: number,
-    xx: 'XX'
-  ): Promise<number | null>;
+  pexpire(key: RedisKey, milliseconds: number, xx: 'XX'): Promise<number>;
 
-  pexpire(
-    key: RedisKey,
-    milliseconds: number,
-    gt: 'GT'
-  ): Promise<number | null>;
+  pexpire(key: RedisKey, milliseconds: number, gt: 'GT'): Promise<number>;
```

Comment:
why is the return type changing here?

Reply:
I think this was typo on my part when I originally implemented `pexpire` was re-checking out interface in original SDK and I think this should be what is now.
https://github.com/redis/ioredis/blob/main/lib/utils/RedisCommander.ts#L5400-L5428

No chance to return undefined instead of number.

Reply:
Added more explicit test as well for this
https://github.com/momentohq/momento-node-ioredis-client/pull/35/commits/8b8472fda8d997f76f74f089ac3e172296092968